kf_lib_data_ingest/network/utils.py

```diff
@@ -0,0 +1,114 @@
+"""
```

Comment:
Should prob move all network related utils from common.misc into network.utils in another PR. 

Things to move
- requests_retry_session
- get_open_api_v2_schema